app/utils/kcidb.py

```diff
@@ -179,6 +179,25 @@ def push_tests(group_id, bq_options, db_options={}, db=None):
         'instance': group[models.BOARD_INSTANCE_KEY],
     }
 
+    file_name = "-".join([group[models.NAME_KEY],
+                          group[models.DEVICE_TYPE_KEY]])
+    files = [".".join([file_name, ext]) for ext in ["txt", "html"]]
+
+    lava_json = "-".join(["lava-json", group[models.DEVICE_TYPE_KEY]])
+    lava_json = ".".join([lava_json, "json"])
+    files.append(lava_json)
+
+    urls = tuple("/".join([
+        STORAGE_URL,
+        build[models.FILE_SERVER_RESOURCE_KEY],
+        file]) for file in files
+    )
```

Comment:
Same here.  I'm duplicating code to build filenames and URLs that should probably be shared with the callback code.

Reply:
I think this one is not _too_ bad because it uses the file_server_resource value from the build document.  So it's not perfect but acceptable imo.

Reply:
OK